README.md

```diff
@@ -2,6 +2,12 @@
 
 A CLI to interact with the Internet Computer App on Ledger Nano S/X devices.
 
+## Important Note
+
+The CLI does NOT support tokens with different than eight decimals at the moment.
```

Comment:
You can use an alert `> [!IMPORTANT]` for that maybe?

See https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
Thanks! Much nice